modules/aggregations/src/internalClusterTest/java/org/elasticsearch/aggregations/pipeline/SerialDiffIT.java

```diff
@@ -66,6 +71,12 @@ public String toString() {
         }
     }
 
+    // TODO: maybe add base class that overwrites nodePlugins(...) for all tests that will be added to this module.
```

Comment:
I will add base test classes in a followup change, so that we will not have to add `AggregationsPlugin` to each test class to will exist in this module.